### ISSUE TITLE
Issue about padding token. 

### ISSUE THREAD

**Issue by tmtmaj** [SKTBrain/KoBERT#36]
안녕하세요. 먼저 좋은 소스 제공해주셔서 감사드립니다!

사용 중에 궁금한 점이 생겨 여쭤보려 글 남깁니다. 

padding token의 설정에 관련한 질문인데요. 

모델 서머리를 보면 word embedding에서 padding_idx를 0으로 설정하는데 따로 제공해주신 vocabulary에서는 [PAD]이 1로 설정되어있습니다. 

model summary
```

BertModel(
  (embeddings): BertEmbeddings(
    (word_embeddings): Embedding(8002, 768, padding_idx=0)
    (position_embeddings): Embedding(512, 768)
    (token_type_embeddings): Embedding(2, 768)
    (LayerNorm): LayerNorm((768,), eps=1e-12, elementwise_affine=True)
    (dropout): Dropout(p=0.1, inplace=False)
  )
...
```

vocab

```
{'[UNK]': 0,
 '[CLS]': 2,
 '[SEP]': 3,
 '[MASK]': 4,
 '[PAD]': 1,
 '!': 5,
 "!'": 6,
 '!”': 7,
...
```

bert model에 input tokens과 함께 padding이 아닌 위치에 attention을 주는 attention_masks가 있어 큰 상관없는 줄 알았는데, 출력값이 아래와 같이 달라져 의문이 생겨 질문드립니다.

```
input_ids = torch.LongTensor([[31, 51, 99], [15, 5, 1]])
input_mask = torch.LongTensor([[1, 1, 1], [1, 1, 0]])
token_type_ids = torch.LongTensor([[0, 0, 0], [0, 0, 0]])
sequence_output, pooled_output = model(input_ids, input_mask, token_type_ids)
sequence_output
```

>tensor([[[-0.2365,  0.2418,  0.2133,  ..., -0.5110, -0.0360,  0.0815],
         [-0.2378,  0.2405,  0.2103,  ..., -0.5120, -0.0353,  0.0812],
         [-0.2377,  0.2407,  0.2104,  ..., -0.5113, -0.0356,  0.0817]],
        ...[[-0.0930, -0.4689, -0.0698,  ..., -0.1506, -0.2966, -0.1554],
         [-0.1028, -0.4727, -0.0740,  ..., -0.1520, -0.2981, -0.1605],
         [**-0.0929, -0.4689, -0.0699,  ..., -0.1507, -0.2966, -0.1554**]]],
       grad_fn=<NativeLayerNormBackward>)

```
input_ids = torch.LongTensor([[31, 51, 99], [15, 5, 0]])
input_mask = torch.LongTensor([[1, 1, 1], [1, 1, 0]])
token_type_ids = torch.LongTensor([[0, 0, 0], [0, 0, 0]])
sequence_output, pooled_output = model(input_ids, input_mask, token_type_ids)
sequence_output
```

>tensor([[[-0.2365,  0.2418,  0.2133,  ..., -0.5110, -0.0360,  0.0815],
         [-0.2378,  0.2405,  0.2103,  ..., -0.5120, -0.0353,  0.0812],
         [-0.2377,  0.2407,  0.2104,  ..., -0.5113, -0.0356,  0.0817]],
       ... [[-0.0930, -0.4689, -0.0698,  ..., -0.1506, -0.2966, -0.1554],
         [-0.1028, -0.4727, -0.0740,  ..., -0.1520, -0.2981, -0.1605],
         [ **0.1585, -0.1319,  0.2887,  ...,  0.2637, -0.2277,  0.2331**]]],
       grad_fn=<NativeLayerNormBackward>)

**Comment by haven-jeon** [comment]
pytorch의 bert config을 정확하게 명시하지 않아서 기본값인 0을 넣어버린 이슈로 보입니다. 
transformers에서 많은 부분 config에서 default parameter로 유추하고 있어서 직접 명시적으로 코딩하지 않는 경우 오류가 발생할 수 있어 이 부분은 고쳐져야 될 이슈네요. 

리포팅 감사합니다.

**Comment by haven-jeon** [comment]
config 수정 했습니다. 확인 부탁드립니다.

**Comment by tmtmaj** [comment]
감사합니다. 모델 embedding에서 padding_idx = 1로 들어가는 것 확인했습니다.
